react/DynamicIframe.tsx

```diff
@@ -3,35 +3,29 @@ import React from 'react'
 import { useRuntime } from 'vtex.render-runtime'
 
 const DynamicIframe: StorefrontFunctionComponent<DynamicIframeProps> = ({
-  srcPrepend,
-  dynamicParams,
-  srcAppend,
+  dynamicSrc,
   width,
   height,
   title,
 }) => {
   const {
     route: { params },
   } = useRuntime()
-  let dynamicSrcValues = ''
-  if (dynamicParams && dynamicParams.length) {
-    dynamicParams.forEach(thisValue => {
-      if (params[thisValue]) {
-        dynamicSrcValues += '/' + params[thisValue]
+  if (dynamicSrc && dynamicSrc.length) {
+    dynamicSrc.split('/').forEach(thisValue => {
+      if (thisValue.indexOf('{') >= 0) {
+        var thisParam = thisValue.replace('{', '').replace('}', '')
+        if (params[thisParam] && dynamicSrc.indexOf(thisValue) >= 0) {
+          dynamicSrc = dynamicSrc.replace(thisValue, params[thisParam])
+        }
```

Comment:
You can replace this code with this:

```js
src.replace(/({[A-z0-1]*})/g, function(match) {
  param = match.replace(/{|}/g, '')
  return params[param]
})
```

Example:
![image](https://user-images.githubusercontent.com/284515/72464214-9a8fa400-37b3-11ea-945f-a1c480d57c2d.png)